data/cameras.xml

```diff
@@ -11608,42 +11608,6 @@
 			</ColorMatrix>
 		</ColorMatrices>
 	</Camera>
-	<Camera make="PENTAX" model="PENTAX K100D" supported="no-samples">
```

Comment:
I think in cases like these ,it would be best to still keep the entry, but with `mode="dng"`,